src/wallet/wallet.h

```diff
@@ -735,7 +735,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * Insert additional inputs into the transaction by
      * calling CreateTransaction();
      */
-    bool FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nChangePosRet, std::string& strFailReason, bool includeWatching);
```

Comment:
It doesn't really matter as long as existing code either fails to compile or behaves the same. Note that to put destChange earlier, you would need to make it a required option.